cmd/export_ledgers.go

```diff
@@ -54,13 +82,62 @@ func printTransformStats(attempts, failures int, printLog bool) {
 
 	results, err := json.Marshal(resultsMap)
 	if err != nil {
-		cmdLogger.Fatal("Could not marshall results: ", err)
+		cmdLogger.Fatal("Could not marshal results: ", err)
 	}
 
-	if printLog {
-		fmt.Println(string(results))
-	} else {
-		cmdLogger.Info(string(results))
+	cmdLogger.Info(string(results))
+}
+
+func generateRunId() string {
+	return uuid.New().String()
+}
+
+func uploadToGcs(credentialsPath, bucket, runId, path string) error {
```

Comment:
One other idea: does it make sense to include the `job_name` in the GCS URI? Curious if you think it would make debugging easier if ever we needed to look at the data. 
Instead of
```
gs://stellar-etl-cli/<run_id>/<filename>
```
Add a directory by `job_name` (using `export-accounts` as example):
```
gs://stellar-etl-cli/export-accounts/<run_id>/<filename>
```

Reply:
I was also thinking more about the output file structure after looking at how it is used in the dags. What do you think of getting rid of run_id and doing something like: 

```
gs://stellar-etl-cli/export-accounts/<filename>
```

or 

```
gs://stellar-etl-cli/accounts/<filename>
```

Reply:
I'm in favor of that. I think it's simpler and makes sense. With that structure would the file names be unique per executed batch? Like:
```
accounts-<run_id>.txt
```
or 
```
<ledger-range>.txt
```
etc

Reply:
After giving this some thought, I decided to push this problem up a layer, and let the airflow code decide how to namespace the data from each run. The resulting behavior is that the `stellar-etl` CLI naively takes the output path provided, and writes the content there. 

So, invoking 

```
stellar-etl export_ledgers -s 100000 -e 100001 -g stellar-etl-cli -o testdata/<run_id>
```

will produce 

```
testdata/<run_id>/100000-10001-accounts.txt
```

which will then be uploaded to

```
gs://stellar-etl-cli/testdata/<run_id>/testdata/<run_id>/100000-10001-accounts.txt
```

The local copy is deleted after that.